### PR TITLE
fix UI

### PR DESCRIPTION
**Pull request by dkracheli**

### CODE REVIEW

File: controllers/accountsController.js [hunk 1]
```diff
@@ -27,12 +28,8 @@ const inviteUser = async (req, res) => {
 };
 
 const getAccount = async (req, res) => {
```
Comment (shaharariel95) [inline controllers/accountsController.js:30]: is it secure?

File: controllers/accountsController.js [hunk 2]
```diff
@@ -27,12 +28,8 @@ const inviteUser = async (req, res) => {
...
-  const user = await User.retrieve({ email: req.params.email });
-
-  if (!user) throw new Error('user does not exists');
-
-  const acc = await Account.retrieve({ _id: user.accountId });
```
Comment (shaharariel95) [inline controllers/accountsController.js:34]: variable name is not understandable, please change to account from acc

File: routes/accountsRouter.js [hunk 3]
```diff
@@ -5,9 +5,9 @@ const accountController = require('../controllers/accountsController');
 const { checkPermission, checkPermissionAdmin } = require('../middleware/validatorService');
 
 accountsRouter.get('/:account/invite/:email', accountController.inviteUser); // done
-accountsRouter.get('/'/* , checkPermissionAdmin*/, accountController.getAccounts); // done
-accountsRouter.get('/:email/', accountController.getAccount); // done
-accountsRouter.put('/edit/'/* , checkPermission */, accountController.editAccount); // done
-accountsRouter.put('/disable/'/* , checkPermission */, accountController.disableAccount);
+accountsRouter.get('/', checkPermissionAdmin, accountController.getAccounts); // done
+accountsRouter.get('/:accountId/', accountController.getAccount); // done
+accountsRouter.put('/edit/', checkPermission, accountController.editAccount); // done
```
Comment (shaharariel95) [inline routes/accountsRouter.js:10]: have you checked that it works?

File: client/addUser.html [hunk 4]
```diff
@@ -92,11 +92,26 @@ <h3 class="welcome-sub-text">Your performance summary this week </h3>
...
             <ul class="nav">
                 <li class="nav-item">
-                    <a aria-controls="ui-basic" aria-expanded="false" class="nav-link" data-bs-toggle="collapse"
-                       href="#ui-basic">
-                        <span class="menu-title">Add User</span>
+                    <a class="nav-link addEmail" href= "addUser.html" aria-expanded="false"
+                       aria-controls="ui-basic">Add User
                     </a>
                 </li>
+                <li class="nav-item">
```
Comment (shaharariel95) [inline client/addUser.html:99]: missing my account page link

File: client/editProfile.html [hunk 5]
```diff
@@ -90,11 +90,26 @@ <h3 class="welcome-sub-text">Your performance summary this week </h3>
...
-                       aria-controls="ui-basic">
-                        <span class="menu-title">Add User</span>
+                    <a class="nav-link addEmail" href= "addUser.html" aria-expanded="false"
+                       aria-controls="ui-basic">Add User
                     </a>
                 </li>
+                <li class="nav-item">
+                    <a class="nav-link addEmail" href="myProfile.html" aria-expanded="false"
+                       aria-controls="ui-basic">My Profile
+                    </a>
```
Comment (shaharariel95) [inline client/editProfile.html:100]: missing my account page link

File: client/myAccount.html [hunk 6]
```diff
@@ -91,11 +91,26 @@ <h3 class="welcome-sub-text">Your performance summary this week </h3>
...
                 <li class="nav-item">
-                    <a class="nav-link" data-bs-toggle="collapse" href="#" aria-expanded="false"
-                       aria-controls="ui-basic">
-                        <span class="menu-title">All Accounts</span>
+                    <a class="nav-link addEmail" href= "addUser.html" aria-expanded="false"
+                       aria-controls="ui-basic">Add User
                     </a>
                 </li>
+                <li class="nav-item">
+                    <a class="nav-link addEmail" href="myProfile.html" aria-expanded="false"
```
Comment (shaharariel95) [inline client/myAccount.html:99]: missing a link to my account page, even if its in the my account html

File: client/myProfile.html [hunk 7]
```diff
@@ -90,17 +90,26 @@ <h3 class="welcome-sub-text">Your performance summary this week </h3>
...
-                       aria-controls="ui-basic">
-                        <span class="menu-title">My Profile</span>
+                    <a class="nav-link addEmail" href= "addUser.html" aria-expanded="false"
+                       aria-controls="ui-basic">Add User
                     </a>
                 </li>
                 <li class="nav-item">
-                    <a class="nav-link" data-bs-toggle="collapse" href="#ui-basic" aria-expanded="false"
-                       aria-controls="ui-basic">
-                        <span class="menu-title">My Account</span>
```
Comment (shaharariel95) [inline client/myProfile.html:101]: missing my account page link

File: client/script/script.js [hunk 8]
```diff
@@ -36,18 +36,20 @@ const generateAccountTable = (table, data) => {
...
-        const option = buttonOption(element.Name, 'myAccount');
-        cell.appendChild(button);
-        cell.appendChild(option);
-      } else if (key === 'Status') {
-        const status = statusStyle(element[key]);
-        cell.appendChild(status);
-      } else {
-        const text = document.createTextNode(element[key]);
-        cell.appendChild(text);
+      if (key !== 'id') {
```
Comment (ofirpeleg) [inline client/script/script.js:39]: is this hidden key?

File: routes/accountsRouter.js [hunk 9]
```diff
@@ -5,9 +5,9 @@ const accountController = require('../controllers/accountsController');
 const { checkPermission, checkPermissionAdmin } = require('../middleware/validatorService');
```
Comment (ofirpeleg) [inline routes/accountsRouter.js:5]: ⚡️

**Review by ofirpeleg — COMMENTED** [review]
looks great! 💥 need to add some check permissions (user role)

**Pull request merged**